src/Organic/Organic.php

```diff
@@ -15,6 +14,9 @@
 const GAM_ID_META_KEY = 'empire_gam_id';
 const SYNC_META_KEY = 'empire_sync';
 
+// The DSN to use before we load client-specific DSNs.
+const DEFAULT_SENTRY_DSN = 'https://e1cf660e5b3947a4bdf7c516afaaa7d2@o472819.ingest.sentry.io/4505048050434048';
```

Comment:
I don't like having this public, but 1) Sentry (questionably) [says it's ok](https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization) and 2) I don't see a way around it unless we provide the DSN in the platform and have them paste it into their WordPress plugin settings (like we do for the API key).

Reply:
Another option: we only get the DSN from the platform, which either returns the client-specific DSN or else the some default DSN. The only disadvantage here is that we don't get any Sentry logging for initializing the plugin.

Reply:
Another option: we only get the DSN from the platform, which either returns the client-specific DSN or else the some default DSN. The only disadvantage here is that we don't get any Sentry logging for initializing the plugin.